packages/react/src/components/Flex/FlexItem.tsx

```diff
@@ -80,9 +80,8 @@ class FlexItem extends UIComponent<Extendable<FlexItemProps>> {
       })
     }
 
-    return _.isNil(children)
-      ? children
-      : applyStyles(React.Children.only(children), styles, classes)
+    if (_.isNil(children)) return children
```

Comment:
we cannot use ternary operator anymore or was it your preference here?

Reply:
It becomes unreadable:

```js
return _.isNil(children)
  ? children
  : applyStyles(
    React.Children.only(children) as React.ReactElement<any>,
    styles, 
    classes,
  )
```